colorpicker/src/main/java/com/fourmob/colorpicker/ColorPickerDialog.java

```diff
@@ -0,0 +1,145 @@
+package com.fourmob.colorpicker;
+
+import ohos.agp.colors.RgbColor;
+import ohos.agp.components.Component;
+import ohos.agp.components.DirectionalLayout;
+import ohos.agp.components.LayoutScatter;
+import ohos.agp.components.ProgressBar;
+import ohos.agp.components.Text;
+import ohos.agp.window.dialog.CommonDialog;
+import ohos.app.Context;
+import ohos.global.resource.NotExistException;
+import ohos.global.resource.WrongTypeException;
+import com.fourmob.colorpicker.util.LogUtil;
+import com.fourmob.colorpicker.util.PreferencesHelper;
+import java.io.IOException;
+
+/**
+ * Class for creating a Color Picker Dialog.
+ */
+public class ColorPickerDialog extends CommonDialog implements ColorPickerSwatch.OnColorSelectedListener {
+    protected RgbColor[] mColors = null;
+    protected int mColumns;
+    protected ColorPickerSwatch.OnColorSelectedListener mListener;
+    private ColorPickerPalette mPalette;
```

Comment:
Sort the member variables based on this order static , public, protected, private